packages/node/__tests__/integration.spec.ts

```diff
@@ -32,6 +32,7 @@ describe('Disable Introspection with plugin', () => {
     })
 
     expect(response.statusCode).toBe(400)
+    expect(response.headers['content-type']).toBe('application/json')
     const body = JSON.parse(response.text)
```

Comment:
This should now be possible?
```suggestion
    const { body } = response
```